uftrace.c

```diff
@@ -1428,6 +1428,10 @@ int main(int argc, char *argv[])
 		ret = 0;
 		goto cleanup;
 	case -3:
+		if (opts.mode) {
+			execlp("man", "man", "uftrace", argv[1], (char *)NULL);
+			abort();
```

Comment:
I don't think we need `abort()` here and maybe it's better to call `exit(1)` instead.  But we used to show the full message for the `--help` option, so it'd be natural to fall back to the old behavior if no manual pages are available.

Reply:
`abort()` here is to show this is unreachable. It doesn't hit the `abort()` even if `No manual entry for ...` error message is shown.

But it may be reachable if `man` command itself is not found in the system, which is very unlikely. Maybe the following is better to cover such case as well.
```c
        case -3:
                if (opts.mode) {
                        execlp("man", "man2", "uftrace", argv[1], (char *)NULL);
                        /* fall through if man command itself is not found */
                }
                if (opts.use_pager)
                        start_pager(setup_pager());
                pr_out(uftrace_usage);
                pr_out(uftrace_help);
                wait_for_pager();
                ret = 0;
                goto cleanup;
```

Reply:
> But we used to show the full message for the --help option, so it'd be natural to fall back to the old behavior if no manual pages are available.

Once the `man` command is executed by `execlp`, then there is no way to go back to the fallback routine. If that is really needed, then we should check whether the proper man page can be found before calling `execlp`. But I think we don't have to make it too much complicated.

Reply:
Oh.. ok.  But sometimes `man` might not be installed.  I admit it's a rare condition. :)